pkg/handlers/tdf.go

```diff
@@ -170,3 +214,77 @@ func (h Handler) InspectTDF(toInspect []byte) (TDFInspect, []error) {
 		return TDFInspect{}, []error{fmt.Errorf("tdf format unrecognized")}
 	}
 }
+
+func readBytesFromFile(filePath string) ([]byte, error) {
+	fileToEncrypt, err := os.Open(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open file at path %s: %w", filePath, err)
+	}
+	defer fileToEncrypt.Close()
+
+	bytes, err := io.ReadAll(fileToEncrypt)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read bytes from file at path %s: %w", filePath, err)
+	}
+	return bytes, nil
+}
+
+func correctKeyType(assertionKey sdk.AssertionKey, public bool) (interface{}, error) {
```

Comment:
Do you think this functionality could be exposed within provided platform lib/SDK functionality instead of recreated within each Go PEP?

Reply:
seems like most go packages that deal with crypto just take in args that are already typed rsa.PrivateKey or rsa.PublicKey, they assume you already have the key and if not they expect you to load it yourself. i think the cli has a bit extra code since the whole assertion object is being passed in as a string which isnt how it would normally be done in a go pep. i assume the pep would just construct he assertion similar to https://github.com/opentdf/platform/blob/7d55a909ac03e5a1234709d9051f906d9efd469e/sdk/tdf_test.go#L400-L425. so TLDR i think the cli is more of an outlier in how its passing in assertions and the extra steps needed to parse them and want to avoid adding specific logic to the sdk just for the cli.